js/main.js

```diff
@@ -0,0 +1,16 @@
+const getRundomNumber = (numberOne, numberTwo) => {
+  if(numberOne > numberTwo) {
+    numberOne = Math.floor(numberOne);
+    numberTwo = Math.ceil(numberTwo);
+    return Math.floor(Math.random() * (numberOne - numberTwo)) + numberTwo;
+  }
+
+  numberOne = Math.ceil(numberOne);
+  numberTwo = Math.floor(numberTwo);
+  return Math.floor(Math.random() * (numberTwo - numberOne)) + numberOne;
+};
+
+const checkMaxLength = (string, maxLength) => string.length <= maxLength;
```

Comment:
The function name can be prefixed with is/has/was/have in methods that return a boolean value (true or false).
Examples: `hasChildren(), isPrime()` etc.